.github/workflows/test.yml

```diff
@@ -13,11 +13,11 @@ jobs:
       fail-fast: false
       max-parallel: 5
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10', 'pypy-3.8-v7.3.7']
-        django-version: ['3.2', '4.0', 'main']
```

Comment:
Don't we need django main in CI?

Reply:
Django main is specified in the `include` part of the build definition matrix so this declaration has been a duplicate :)